packages/backend/src/index.ts

```diff
@@ -66,6 +66,7 @@ import lighthouse from './plugins/lighthouse';
 import linguist from './plugins/linguist';
 import devTools from './plugins/devtools';
 import nomad from './plugins/nomad';
+import jiraDashboard from './plugins/jiraDashboard';
```

Comment:
Backstage is using a "-" notation in case of many words, so in this case it could be jira-dashbaord